### PR TITLE
#1735 Refactor pipeline new/edit page

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1735)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
rety manager-it

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-it

File: ohara-manager/client/src/components/pages/PipelinePage/PipelineNewPage/Operate.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      streamsPromises = streams.map(stream => streamApi.stopStreamApp(stream));
+    } else {
+      connectorPromises = _connectors.map(connector =>
+        connectorApi.stopConnector(connector),
+      );
+
+      streamsPromises = streams.map(stream => streamApi.stopStreamApp(stream));
+    }
+
+    return Promise.all([...connectorPromises, ...streamsPromises]).then(
```
Comment (wu87988622) [inline ohara-manager/client/src/components/pages/PipelinePage/PipelineNewPage/Operate.js]: all promise return result ?

Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/PipelineNewPage/Operate.js]: Yep. The resolved values

**Comment by konekoya** [comment]
The failed test is not related

**Pull request merged**